sky/engine/core/dart/painting.dart

```diff
@@ -165,7 +165,8 @@ class Path extends NativeFieldWrapperClass2 {
   /// Adds a new subpath that consists of the straight lines and
   /// curves needed to form the rounded rectangle described by the
   /// argument.
-  void addRRect(RRect rrect) native "Path_addRRect";
+  void addRRect(RRect rrect) => _addRRect(rrect._value);
+  void _addRRect(Float32List rrect) native "Path_addRRect";
```

Comment:
Grabbing `rrect._value` in Dart before switching to C++ is faster than first switching to C++ and then using `Dart_GetField` to grab `_value`.

Using Float32List versus Float64List doesn't matter here because neither has the same memory layout as Skia.  For drawVertices and friends, the memory layout matters because we can reinterpret_cast the memory to match what Skia wants, which saves us a memcpy.